projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-button.php

```diff
@@ -202,7 +202,11 @@ function sharing_process_requests() {
 		}
 	}
 }
-add_action( 'template_redirect', __NAMESPACE__ . '\sharing_process_requests', 9 );
+
+// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Only checking for the data being present.
+if ( isset( $_GET['share'] ) ) {
```

Comment:
If we check for this before the hook, should we remove that check from the actual `sharing_process_requests` function?

Reply:
I think we could keep it there as well
The hook uses the data from `$_GET['share']`, and it's always helpful to check the array element for existence before using it.